src/apis/blockchain_api.ts

```diff
@@ -17,6 +19,14 @@ export const BlockchainAPIErrors = {
                          with hash ${txHash}.`,
 };
 
+/**
+ * The following default timeout is provided to the IntervalManager when awaiting mined
+ * transactions. The value is represented in milliseconds.
+ *
+ * @type {number}
+ */
+export const DEFAULT_TIMEOUT_FOR_TX_MINED = 60000;
```

Comment:
Can we add a bit more color as to why this value was chosen? What is it's significance

Reply:
Hmmm, it may be arbitrary - it's what we currently use in Plex; that's where i got it from.